intrastat_product/wizards/intrastat_result_view.py

```diff
@@ -9,6 +9,6 @@ class IntrastatResultView(models.TransientModel):
     _name = "intrastat.result.view"
     _description = "Pop-up to show errors on intrastat report generation"
 
-    note = fields.Text(
+    note = fields.Html(
```

Comment:
This requires a migration script to convert the plain text to html. There's an openupgradelib method for it. On pre, copy the column to preserve the old content untouched, and on post, do the conversion.

Reply:
Ok then